install/kubernetes/helm/istio-remote/templates/endpoints.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.global.remotePilotCreateSvcEndpoint }}
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: istio-pilot
+  namespace: {{ .Release.Namespace }}
+subsets:
+- addresses:
+  - ip: {{ .Values.global.remotePilotAddress }}
+  ports:
+  - port: 15003
```

Comment:
I don't think we have those ports any more.

Reply:
The port list is consistent with the main cluster's pilot chart:  https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/pilot/templates/service.yaml